### PR TITLE
Add diagrams

### PR DESCRIPTION
**Pull request by oblute**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by oblute** [comment]
@rluria14 @ndmaxar agree to be maintainers?

File: recipes/diagrams/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python-graphviz >=0.13.2,<0.14.0
+    - jinja2 >=2.10,<3.0
+    - python
+    - contextvars
```
Comment (marcelotrevisani) [inline recipes/diagrams/meta.yaml:25]:
```suggestion
    - contextvars >=2.4,<3.0  # [py>=36 and py<37]
```

File: recipes/diagrams/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dcc03eccdfbbfb25e3804f0e13c5948292ba81b4ccf6f3a863b9467b7b5fe1ea
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not py36]
```
Comment (marcelotrevisani) [inline recipes/diagrams/meta.yaml:15]:
```suggestion
  skip: True  # [py2k]
```

**Comment by marcelotrevisani** [comment]
If you want to, next time you can give a go to [grayskull :skull: ](https://github.com/marcelotrevisani/grayskull/) to generate recipes for conda looking for packages on pypi :smiley:

File: recipes/diagrams/meta.yaml (hunk 2 above)
Comment (oblute) [reply recipes/diagrams/meta.yaml]: @marcelotrevisani would this be necessary given that conda-forge no longer supports python 2.7 anyway?

Comment (marcelotrevisani) [reply recipes/diagrams/meta.yaml]: yeap, in that sense it is not necessary
I just added another suggestion which is specified in the metadata of this package, but I don't know if it is really necessary
or do you want to create a package for just python 3.6 ?

Comment (oblute) [reply recipes/diagrams/meta.yaml]: @marcelotrevisani thank you, the selector you added helped get py37 to work!

**Comment by oblute** [comment]
@marcelotrevisani i think those changes are good to go :)

**Pull request merged**